### PR TITLE
Add new image type:vhd/gce support in bootc image builder

### PR DESCRIPTION
**Pull request by chunfuwen**
Add new image type:vdh/gce support in bootc image builder
vhd/gce are newly introduced support disk image type, and is only supported on upstream

### CODE REVIEW

File: provider/bootc_image_builder/bootc_image_build_utils.py [hunk 1]
```diff
@@ -838,6 +838,21 @@ def cleanup_aws_ami_and_snapshot(params):
     aws_utils.delete_aws_ami_snapshot_id(params)
 
 
+def convert_vhd_to_qcow2(params):
+    """
+    Convert vhd disk format into qcow2
+
+    @param params: one dictionary wrapping various parameter
```
Comment (nanli1) [inline provider/bootc_image_builder/bootc_image_build_utils.py:845]: Missing    return converted_image_path  instruction here.

Comment (chunfuwen) [reply provider/bootc_image_builder/bootc_image_build_utils.py:936]: updated

**Comment by chunfuwen** [comment]
@dzhengfy 
@smitterl 

Please help review this PR if you are free

**Review by nanli1 — APPROVED** [review]

**Comment by chunfuwen** [comment]
Let's merge it since it has been pending for long time

**Pull request merged**